dat/Atari - ST.dat

```diff
@@ -1579,6 +1579,12 @@ game (
 )
 
 game (
+	name "Stone Age (Europe) (Amiga + ST)"

```

Comment:
We could put them in a separate dat, that way they get combined and you don't need to keep modifying the main DATs like this. I think that would probably be a more sustainable route.